test/TensorFlow/sends_recvs.swift

```diff
@@ -392,3 +392,63 @@ public func testRecvsInALoop() {
   // This one should not be a send.
   _hostOp(a.toHost())
 }
+
+let globalIterator: ResourceHandle =
+  #tfop("Iterator", shared_name: "foo", container: "bar",
+        output_types: [Float.self], output_shapes: [TensorShape()])
+let globalNextIterator: ResourceHandle =
+  #tfop("IteratorGetNextAsOptional", globalIterator,
+        output_types: [Float.self], output_shapes: [TensorShape()])
+
+public func resourceHandlesCanBeSentOrReceived() {
+  // The following constant is a workaround to extend the deabstraction scope so
```

Comment:
do we still need the *workaround* both at the beginning and the end of the test?

or are they intended so that we are testing sends/recvs?

Reply:
The workaround at the beginning may be removed once we convert arg tensors to swift-TF sends. The workaround at the end needs to be there to test the receive. Otherwise, the result will become the output value of the tensor program.

Reply:
In the second case, please rename "workaround" to a phrase that indicates it's intentional/expected.